src/net.cpp

```diff
@@ -2838,7 +2839,10 @@ bool CConnman::NodeFullyConnected(const CNode* pnode)
 void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg&& msg)
 {
     size_t nMessageSize = msg.data.size();
-    LogPrint(BCLog::NET, "sending %s (%d bytes) peer=%d\n",  SanitizeString(msg.m_type), nMessageSize, pnode->GetId());
+    auto sanitizedType = SanitizeString(msg.m_type);
+
+    LogPrint(BCLog::NET, "sending %s (%d bytes) peer=%d\n", sanitizedType, nMessageSize, pnode->GetId());
+    TRACE4(net, push_message, sanitizedType.c_str(), msg.data, nMessageSize, pnode->GetId());
```

Comment:
I'm not sure we want to pass the sanitized packet type to the probe here or simply the raw data.